OPTIONS.md

```diff
@@ -0,0 +1,46 @@
+# VarTable Options
+
+## (Required)
+**--input-vcf**		input VCF file
+
+*	Do we need to differentiate between NGS_Mapper VCF (every position) versus lofreq VCF (only variants)?*
+
+**--input-bam**		input BAM file
+
+*	Do we assume BAM index file is in the same directory as BAM or should we include it as an option?
```

Comment:
Don't need to pass the index file as input, it will always be in the same directory.
However, it is required and we should check for it elsewhere